### ISSUE TITLE
Received an unexpected response from the Automatic1111 Stable Diffusion server

### ISSUE THREAD

**Issue by dmitrybaltin** [benrugg/AI-Render#122]
### Describe the bug

I try to use AI-render addon together with Automatic1111 deployed in RunPod services using jupiter notebook 
[fast-stable-diffusion](https://github.com/TheLastBen/fast-stable-diffusion)
It's the way recomended by [Runpod](https://blog.runpod.io/stable-diffusion-ui-on-runpod/?utm_term=&utm_campaign=Serverless+GPU&utm_source=adwords&utm_medium=ppc&hsa_acc=4558579452&hsa_cam=20156995097&hsa_grp=&hsa_ad=&hsa_src=x&hsa_tgt=&hsa_kw=&hsa_mt=&hsa_net=adwords&hsa_ver=3&gclid=CjwKCAjw-7OlBhB8EiwAnoOEkxe2hlFSm54tM7EcqmMyQY0PTrR8VaMrtkItS-siGc4SOoFpsMx2jRoCYrEQAvD_BwE)

The connection is established and the request is successull sent but there is an exception occurses in the file automatic1111_api.py in the following string of handle_success() function:
```python
response_obj = response.json()
``` 
All the code before the wrong string are executed correctly including the checking the status in the string 
```python
if response.status_code == 200:
``` 
Also I see in the jupyter notebook with SD that the SD works and generates the picture.

I dumped the response.content object to the file. It looks like a binary data and I can't understand it's format by visual seeing.
The size of file is ~350-400 kb and it 's similar the png or jpg file of the size 512x768 witch I try to generate. I attached it here. (I packed it to zip because github don't want to download the file of unknown format)

It's the first time when I see the 'requests' library so I can suppose two reasons of the bug (Maybe my supposings is wrong):
1. The reult is in binary format but not is a text as the AI Renderer addon expects.
2. The result is encrypted (but in this case it's strange that the automatic successfully get the input data)

In the both cases I don't know what to do and to fix it.
Maybe there ara any setting of output data in automatic1111 but I can't find it.

[dump.zip](https://github.com/benrugg/AI-Render/files/12013870/dump.zip)

### To reproduce

The bug is 100% reproducible

1. Start Automatic1111 on Runpod
2. Launch Blender, open the blend file, switch to render mode.
3. Setup the settings of AI Renderer Addon inpreference: choose local automatic1111, enter the URL witch is always new after ralaunching Automatic, usially it looks like https://rhft3k44x3eun0-3001.proxy.runpod.net/, port is not required (i tryed to set the port 7860 but it does not work)
4. Press F12 to render and whait several seconds

### Error log

[BlenderConsoleLog.txt](https://github.com/benrugg/AI-Render/files/12014269/BlenderConsoleLog.txt)

### Environment

- Blender version - 3.3.1: 
- AI Render version - 0.9.0 (last version from Github at 2023-07-03: 
- Operating system - Window 11

**Comment by benrugg** [comment]
Ah, interesting. I don't think I've seen anyone successfully use AI Render with Automatic1111 running on a notebook. (Or if they have, I haven't heard about it). 

I will look into this, but it might be a while before I can get to it!

**Comment by dmitrybaltin** [comment]
May be may way is wrong.
Really it's a little strange that this run pod environment deploys Jupyter and then jupiter deploys automatic 1111)

Maybe somebody can show me simple and correct way how to deploy automatic 1111 to server in Internet. 

I like to use remote servers and cloud services for heavy tasks because it's the way for me to work anywhere using light notebook. And it's cheap way - I usually spend not more 10-20$ for different services. So I can't and I don't plan to deploy SD to my own computer.

Earlier I use Colab for running SD, but some month ago Google broken it, so I found the method using Run pod. It's not free but not expensive, and also it's simple, stable and predictable. And it's cheaper then Dream Studio and I can customise it using automatic 1111.

**Comment by benrugg** [comment]
Yeah, it's a really convenient and inexpensive way to do it, for sure! For what it's worth, you can still do it on the Google Colab notebooks for about the same price you mentioned. (But I'm not sure if anyone has gotten AI Render working with that, either)

**Comment by dmitrybaltin** [comment]
The free Collab account is not working at the moment.
As for the paid account. I haven't tried it, but I've read bad reviews. They say it's not cheap because Google counts certain "computing units".
When I use Runpad, I can choose the configuration and pay for the time spent.
But Google calculates "units", and as I read in the reviews, 100 units by default is really an insignificant amount, and it can be spent in an hour.
Although I haven't actually tried it myself)

**Comment by benrugg** [comment]
Gotcha. Ok, that's good to know!
